src/main/java/org/prebid/server/validation/RequestValidator.java

```diff
@@ -206,25 +207,63 @@ private static String format(BigDecimal value) {
      * Validates {@link ExtRequestTargeting}.
      */
     private static void validateTargeting(ExtRequestTargeting extRequestTargeting) throws ValidationException {
+        validateExtPriceGranularity(extRequestTargeting.getPricegranularity(), null);
+        validateMediaTypePriceGranularity(extRequestTargeting.getMediatypepricegranularity());
+
+        final Boolean includeWinners = extRequestTargeting.getIncludewinners();
+        final Boolean includeBidderKeys = extRequestTargeting.getIncludebidderkeys();
+        if (Objects.equals(includeWinners, false) && Objects.equals(includeBidderKeys, false)) {
+            throw new ValidationException("ext.prebid.targeting: At least one of includewinners or includebidderkeys"
+                    + " must be enabled to enable targeting support");
+        }
+    }
+
+    /**
+     * Validates {@link ExtPriceGranularity}.
+     */
+    private static void validateExtPriceGranularity(JsonNode priceGranularity, String type)
```

Comment:
Can we use `org.prebid.server.proto.openrtb.ext.response.BidType` here instead of `String` for type?